rllib/core/rl_trainer/rl_trainer.py

```diff
@@ -468,6 +507,7 @@ def get_parameters(self, module: RLModule) -> Sequence[ParamType]:
         Returns:
             The parameters of the module.
         """
+        # TODO (Kourosh): Make this method a classmethod
```

Comment:
dude, can you say why we should make this method a classmethod?
get_parameters() doesn't seem like a method that should be classmethod.

Reply:
This function's purpose is to get the parameters of a module based on what the underlying framework is. For example for torch it should be `.paramters()` of the module. For tf it will be `.trainable_variables` of the module. So it def. does not need to be a instance method. In JAX it will be something else.

Reply:
add more explanation to the TODO..

Reply:
Probably need a better function name … just saying …